util/config/config.go

```diff
@@ -95,6 +95,10 @@ var (
 		heights: []uint32{133400, 0},
 		values:  []int{1024, MaxInt},
 	}
+	BucketsLimit = HeightDependentInt{ // ChangePoin for txn payload Subscribe.Meta
+		heights: []uint32{133400, 0},
```

Comment:
133400 is a invalid block number.

Reply:
yeah, need to change this to some future block height when merging